controllers/notifyController.php

```diff
@@ -148,17 +148,50 @@ private function getMerchantOrder()
     }
 
     /**
-     * @throws NoIdentificationException
+     * @throws MerchantOrderNotFoundException
      */
-    private function getPagantisOrderId()
+    private function verifyOrderConformity()
```

Comment:
It is done inside setPagantisOrderId function

Reply:
moved it to getPagantisOrderId